src/StreamJsonRpc/ProxyGeneration.cs

```diff
@@ -79,6 +79,9 @@ internal static TypeInfo Get(TypeInfo serviceInterface, bool disposable)
 
                 VerifySupported(!serviceInterface.DeclaredProperties.Any(), Resources.UnsupportedPropertiesOnClientProxyInterface, serviceInterface);
 
+                var methodNameTransformPropertyGetter = typeof(JsonRpcProxyOptions).GetRuntimeProperty(nameof(JsonRpcProxyOptions.MethodNameTransform)).GetMethod;
+                var funcOfStringStringInvoke = typeof(Func<string, string>).GetRuntimeMethod(nameof(JsonRpcProxyOptions.MethodNameTransform.Invoke), new Type[] { typeof(string) });
```

Comment:
Should we promote these to `static readonly` fields as we have done already for other members?